addons/mail/tests/test_user_modify_own_profile.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
this test is very specific, can we move it to the `test_mail` module ? 
(to not bloat mail too much)